### ISSUE TITLE
[a11y] 9.2.2.1 - time limitations adjustable

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5390]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
# General notes
**Note:** pages without activity in a certain time frame are reloaded. - Start page "Alle Dateien" will be displayed. This needs to be investigated. Is there any option to extend the session?

## Status: fulfilled
- **for all pages**

**Comment by pascalwengerter** [comment]
Tracked in #4664, closing this since it's a question and doesn't seem to be a hard requirement to be fulfilled from an a11y (audit) point of view. Feel free to re-open if I'm mistaken
